frontend/server/src/Controllers/Contest.php

```diff
@@ -3414,6 +3414,51 @@ public static function apiUsers(\OmegaUp\Request $r): array {
         ];
     }
 
+    /**
+     * Searchs users in contest
+     *
+     * @omegaup-request-param mixed $query
+     * @omegaup-request-param mixed $contest_alias
+     *
+     * @return list<array{label: string, value: string}>
+     */
+    public static function apiSearchUsers(\OmegaUp\Request $r): array {
+        $r->ensureIdentity();
+
+        \OmegaUp\Validators::validateStringNonEmpty(
+            $r['contest_alias'],
+            'contest_alias'
+        );
+
+        $contest = self::validateContestAdmin(
+            $r['contest_alias'],
+            $r->identity
+        );
+
+        $param = '';
+        if (is_string($r['query'])) {
+            $param = $r['query'];
+        } else {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterEmpty',
+                'query'
+            );
+        }
+
+        $users = \OmegaUp\DAO\ProblemsetIdentities::searchUsers(
+            $param,
+            intval($contest->problemset_id)
+        );
+        $response = [];
+        foreach ($users as $user) {
+            $response[] = [
+                'label' => $user['username'],
```

Comment:
```suggestion
                'label' => $user['name'] ?? $user['username'],
```

esto implica que también hay que obtener el nombre desde el DAO.